src/features/homePage/HomePage.tsx

```diff
@@ -1,5 +1,13 @@
+import { Loader } from '../Loader';
+
+Loader;
```

Comment:
Is this necessary?

Reply:
It's interesting how it got pass eslint/prettier/husky :)